pkgs/development/python-modules/azure-storage-blob/default.nix

```
@@ -0,0 +1,28 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, azure-common
+, azure-storage-common
+}:
+
+buildPythonPackage rec {
+  pname = "azure-storage-blob";
+  version = "1.4.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "65ebe2e54460566c2077c6b3773a2a0623eabc7b95602010cb51b84077087fda";
+  };
+
+  propagatedBuildInputs = [
+    azure-common
+    azure-storage-common
+  ];
+
+  meta = with stdenv.lib; {
+    description = "Client library for Microsoft Azure Storage services. Contains the blob service APIs";
```

```suggestion
    description = "Client library for Microsoft Azure Storage services ontaining the blob service APIs";
```
Please do the same for the other packages.